pkg/JuliaExperimental/gap/numfield.g

```diff
@@ -288,8 +288,7 @@ InstallMethod( ContextGAPNemo,
     if Length( indetnames ) = 1 then
       names:= GAPToJulia( indetnames[1] );
     else
-      names:= Julia.Base.convert( JuliaEvalString( "Vector{String}" ),
-                                  GAPToJulia( indetnames ) );
+      names:= JuliaEvalString( "Vector{String}" )( GAPToJulia( indetnames ) );
```

Comment:
Off-topic remark: For production code, the value `JuliaEvalString( "Vector{String}" )` should probably be computed only once and stored in a global variable.

Reply:
We have already quite a few of these Julia types in global GAP variables (`_JL_Vector_Bool`, `_JL_Vector_Any`, `_JL_Dict_Symbol`), this pattern can be continued.
In order to (at least) avoid the `JuliaEvalString` call, we could also provide a more programmatic approach: If GAP.jl defines `typeconstruct(T1::Type, T2::Type) = T1{T2}` then one can write `Julia.GAP.typeconstruct( Julia.Vector, Julia.Int )` in a GAP session, and gets the Julia type `Vector{Int64}`; would something in this direction be useful?

Reply:
yeah, that would be useful. Ideally it should also deal with multiple args as in  e.g. `Dict{String, Int}` or `Array{Int, 3}`. 

I actually thought about installing a method for `ELMS_LIST` / `\{\}` to make this convenient to access from GAP code (so you could write `Julia.Vector{Julia.String}` or `Julia.Dict{[Julia.String, Julia.Int]}` but currently GAP insists on the argument being a list of positive integers. I actually have a PR in the works for GAP to lift that limitation, but I am not yet sure we really want it (but I'd like to experiment with it, and it might actually be interesting for other applications in GAP, so I figured "why not try it").